python/pdo/contract/replication.py

```diff
@@ -111,26 +124,38 @@ def __replication_manager__():
                 replication_request.mark_as_completed(response.call_back_after_replication)
                 continue
 
-            #ensure that the worker threads and queues are in place for services associated with this replication request
+            #ensure that the worker threads and queues are initialized
             for service_id in replication_request.service_ids:
                 if __replication_workers_executor__.get(service_id) is None:
                     __set_up_worker__(service_id)
+                else:
+                    #worker threads were initialized in a previous replciation task, ensure that they are still running, else mark
+                    #the corresponding replication service as one to ignore
```

Comment:
would you wrap the comments at a consistent number of columns (around 100 columns), and there is a spelling error in the comment "replciation"